.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
+  exclude:
```

Comment:
@coderabbitai Provide a complete numbered list of all merged PRs for milestone 1.19.0 enhancement label, ensuring that all results are captured by handling pagination properly. Put the results under heading features. Each result is formatted as #PR number - PR title

Provide a complete numbered list of all merged PRs for milestone 1.19.0 bug label, ensuring that all results are captured by handling pagination properly. Put the results under heading bug fixes. Each result is formatted as #PR number - PR title

Reply:
Try again